eng/scripts/Language-Settings.ps1

```diff
@@ -119,9 +119,11 @@ function Get-javascript-PackageInfoFromPackageFile ($pkg, $workingDirectory)
 }
 
 function Get-javascript-DocsMsMetadataForPackage($PackageInfo) { 
+  $docsReadmeName = Split-Path -Path $PackageInfo.DirectoryPath -Leaf
```

Comment:
Thanks for the chat. My only concern is that there's some existing package for which the directory name does not match the pattern we've laid out. For example, the package name is `@azure/package` and the directory is `package-folder-name` and the `.md` filename that gets generated switches from `package-readme.md` to `package-folder-name-readme.md` and we stop updating the readme document that the ToC references and instead update this other file that a user is very unlikely to reach. 

If that is the case, we can mitigate with this pattern: 

```powershell
# Default case
$docsReadmeName = $PackageInfo.Name -replace "^@azure/" , ""
if ($PackageInfo.Name.StartsWith('@azure-rest/')) { 
  # Define naming convention for REST packages
  $docsReadmeName = Split-Path -Path $PackageInfo.DirectoryPath -Leaf
}
```

However, I prefer the diff as is if it doesn't break readme file naming for docs we've already published because it avoids us having to bury special case business logic deep in our code.